regression/pages/lms/login_lms.py

```diff
@@ -0,0 +1,38 @@
+from bok_choy.promise import EmptyPromise
+from edxapp_pages.lms.login import LoginPage
+from edxapp_pages.lms.dashboard import DashboardPage
+from regression.pages.lms import BASE_URL
+
+
+class LmsLogin(LoginPage):
+    """
+    This class is an extended class of LoginPage,
+    where we add methods that are different or not used in LoginPage
+    """
+    url = BASE_URL + '/login'
+
+    def is_browser_on_page(self):
+        return self.q(css='.action.action-primary.action-update.js-login.login-button').present
```

Comment:
Should this be `.visible` instead? That'll be a stronger assertion if it applies.